### ISSUE TITLE
GPU fans randomly stopped, GPU temps up to 90°C and sluggish DE

### ISSUE THREAD

**Issue by freak2fast4u** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#428]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game / Application [Required]

Irrelevant

### Game Platform [Required]

Irrelevant

### Other game platform

Irrelevant

### Processor / Processor Number [Required]

Ryzen 7 7800X3D

### Graphic Card [Required]

Intel Arc A770 16GB LE

### GPU Driver Version [Required]

- [ ] 31.0.101.4578
- [X] 31.0.101.4577

### Rendering API [Required]
Irrelevant

### Windows Build Number [Required]

- [ ] Windows 11 22H2
- [ ] Windows 11 21H2
- [X] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Other Windows build number

19045.3324

### Intel System Support Utility report

[igcit_ssu.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/12329999/igcit_ssu.txt)

### Description and steps to reproduce [Required]

Computer was on for about 30 minutes, and all of a sudden the desktop starts to get sluggish. I check the CPU, which wasn't doing much (10% usage). I open the computer case, the GPU fans were stopped !!

Some possibly noteworthy information : 
- I have Afterburner/RTSS enabled at all times (latest versions for each).
- The fan configuration was set to use a custom fan curve, and I had no issues so far using this configuration until now.

I go to check Arc Control and notice this (no fan speed indication) : 
![arccontrol_no_fans](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/3486369/aa84d60b-82cc-4457-be10-b45de07feeae)

Then I switch the fans to automatic and notice a new "tile" appear on the left, with the fan speed : 
![arccontrol_fans](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/3486369/550e706b-961a-4a06-a613-16794ca543ff)

Somehow Arc Control lost track of the GPU fans all together ...

I resolved the situation short-term by setting the fan speed to "automatic", and the fans ramped right up to 2500rpm and allowed me to finish what I was doing.

### Game graphic quality [Required]

Irrelevant

### Game resolution [Required]

Irrelevant

### Game VSync [Required]

Irrelevant

### Game display mode [Required]

Irrelevant

### Detailed game settings [Required]

Irrelevant

### Device / Platform name

Irrelevant

### Crash dumps [Required, if applicable]

Irrelevant

### Save game

Irrelevant

**Comment by freak2fast4u** [comment]
GPU-Z (v-bios/ifwi version : 20.0.1053, unreported by SSU) ![gpuz_freak2fast4u](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/3486369/31a85bf0-b01b-452f-a0e6-d0c0f42e8b35)

**Comment by freak2fast4u** [comment]
ArcControl perf settings were as follows. The curve was lost when switching to auto, so I re-did it OTOH : 
![ArcControlPerfSettings_freak2fast4u](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/3486369/70343eb8-511f-48bc-8cd2-327cb15e2171)

**Comment by freak2fast4u** [comment]
I heavily edited my initial description because the problem reappeared in a seemingly unspecific and totally unrelated situation, just today.

**Comment by Karen-Intel** [comment]
Heey @freak2fast4u a couple questions:
- What is the ARC Control version you're using? 
- We recommend to update your computer drivers using DDU in safe mode (Latest version is v[18.0.6.5](https://www.guru3d.com/files-details/display-driver-uninstaller-download.html)) please give it a go. If you have done it already just confirm the driver version in device manager.
- Please make sure to have your system updated too. I can see your BIOS is outdated.[ There's a new version for your board](https://www.asus.com/motherboards-components/motherboards/tuf-gaming/tuf-gaming-b650-plus-wifi/helpdesk_bios/?model2Name=TUF-GAMING-B650-PLUS-WIFI) that was released this month.

I will be monitoring this case and my system (A750) but for the time being I have not seen the fans stop randomly. 
Will be on the lookout.

Karen

**Comment by freak2fast4u** [comment]
I flashed bios version 1636 a few days back already (ssu reports mentions this).
ArcControl version 1.70.5084.2, packaged with driver 31.0.101.4577 (currently installed after a clean removal with DDU).
No new trouble so far after not doing anything in particular since my initial report, I'll report back if it happens again.

Thanks for your interest :)

**Comment by freak2fast4u** [comment]
Thanks, I'm testing 101.4644 as we speak and will report back in a few days with the verdict. 
VBIOS also updated as expected by this driver package to version 20.0.1064.

**Comment by freak2fast4u** [comment]
Well, I'm sad to report the problem occured again, just now. All I'm doing is watching videos on youtube using firefox. Wierd thing is, I opened Arc Control to check, and as I was doing a screenshot the fans started going again, and then stopped again, like 10s later. Here goes anyway :
![image](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/3486369/c5219412-44fb-41b0-8335-d2c06b3d29ce)


As you can see, the "Vitesse du ventilateur" tile and label are absent, and the GPU temp is close to 90°C again.

This is using intel arc driver 101.4644. I properly used DDU in safe mode to remove everything "intel gpu" related, and then installed the new driver package.

**Comment by Karen-Intel** [comment]
Hey @freak2fast4u we got a couple updates for fan behavior since driver v.4824, could you check and please let us know?

TY

Karen

**Comment by freak2fast4u** [comment]
Hello @Karen-Intel, thanks for the heads-up ! The initial problem I was having (fans randomly stopping) hasn't resurfaced ever since I installed a version slightly later than 4644 (which still had this problem for me). Possibly 4669, which is the earliest I kept in my downloads folder.

AFAICT, this issue has been fixed for a while.

However, since you mention a very recent version having fixes regarding fan behavior, I'll be on the lookout for any changes in that regard.

I will open a new issue if something else comes up :)

Thanks again !
